src/handlers/shared/check-org-member.ts

```diff
@@ -0,0 +1,9 @@
+import { Context } from "../../types";
+
+export async function isUserMember(context: Context, user: string) {
```

Comment:
Could have a more explicit name, showing that it checks for the user's organization

Reply:
I'll replace with this name `get-user-role-in-org`. Any better suggestions please?